operators.js

```diff
@@ -283,8 +283,9 @@ function or(...args) {
   }
 }
 
-function live() {
-  return (ops) => updateMeta(ops, 'live', true)
+function live(opts) {
+  if (opts && opts.old) return (ops) => updateMeta(ops, 'live', true)
```

Comment:
Actually, because this `meta.live` is only internal, it would be good to rename these:

- live => liveAndOld
- liveOnly => live (because this is the default case `live()` and the semantics of the default case is "live only")